### PR TITLE
Component fixup

### PR DESCRIPTION
**Pull request by joel-hamill**

### CODE REVIEW

File: 1.7/overview/components.md [hunk 1]
```diff
@@ -1,8 +1,11 @@
 ---
-post_title: "An Introduction to DC/OS Components"
+post_title: An Introduction to DC/OS Components
```
Comment (pyronicide) [inline 1.7/overview/components.md]: Needs the quotes.

File: 1.7/overview/components.md [hunk 2]
```diff
@@ -53,7 +49,7 @@ KillMode=mixed
 ```
 
 ## Cluster ID Service
-The cluster-id service allows us to generate a UUID for each cluster. We use this ID to track cluster health remotely (if enabled). This remote tracking allows our support team to better assist our customers.
+The Cluster ID service allows us to generate a UUID for each cluster. If enabled, we can then use this ID to track cluster health remotely. This remote tracking allows our support team to better assist customers.
```
Comment (pyronicide) [inline 1.7/overview/components.md]: Telemetry, not track.

File: 1.7/overview/components.md [hunk 3]
```diff
@@ -68,7 +64,7 @@ ExecStart=/bin/sh -c "/opt/mesosphere/bin/python -c 'import uuid; print(uuid.uui
 ```
 
 ## Cosmos Service
-The Cosmos service is our internal packaging API service. You access this service everytime you run `dcos package install...` from the CLI. This API allows us to deploy DC/OS packages from the DC/OS universe to your DC/OS cluster.
+The Cosmos service is our internal packaging API service. You access this service each time that you run the `dcos package install` command from the CLI. This API allows us to deploy DC/OS packages from the [DC/OS universe](/docs/1.7/usage/services/repo/) to your DC/OS cluster.
```
Comment (pyronicide) [inline 1.7/overview/components.md]: Move the link to the bottom please.

File: 1.7/overview/components.md [hunk 4]
```diff
@@ -53,7 +49,7 @@ KillMode=mixed
 ```
 
 ## Cluster ID Service
-The cluster-id service allows us to generate a UUID for each cluster. We use this ID to track cluster health remotely (if enabled). This remote tracking allows our support team to better assist our customers.
+The Cluster ID service allows us to generate a UUID for each cluster. If enabled, we can then use this ID to monitor cluster health remotely. This remote tracking allows our support team to better assist customers.
```
Comment (pyronicide) [inline 1.7/overview/components.md]: Telemetry, not tracking.

**Comment by pyronicide** [comment]
@sascala looking pretty good. Can you rebase/merge as well?

**Comment by sascala** [comment]
@joel - would you mind doing the rebase? I'm having a hard time resolving the merge conflicts because it's not clear which ones are your edits. I can guess, but...

File: 1.7/overview/components.md (hunk 1 above)
Comment (joel-hamill) [inline 1.7/overview/components.md]: @pyronicide why? i don't see this in other .md files.

**Comment by pyronicide** [comment]
@joel-hamill because the `/` gets interpreted oddly and it breaks the build process.

**Pull request merged**